src/pages/steps/CalendarStep/FixedDays.tsx

```diff
@@ -106,8 +106,8 @@ export const FixedDays = ({
           spacing={3}
           id={`calendar-step-fixed`}
           minDate={new Date()}
-          dateStart={startDate ? new Date(startDate) : undefined}
-          dateEnd={endDate ? new Date(endDate) : undefined}
+          dateStart={new Date(startDate)}
+          dateEnd={new Date(endDate)}
```

Comment:
What if startDate is undefined the Date constructor will throw an error?
https://github.com/cultuurnet/udb3-frontend/commit/a02c1a5a1881bf7a1a1ea0160fc9478287ded244

Reply:
The `startDate` and `endDate` will always be a valid date because they get a default value of today by `createInitialContext`.
The changes on this line make sure the startDate and endDate from the api can only be undefined or a Date:
https://github.com/cultuurnet/udb3-frontend/pull/664/files#diff-5984f8de41ce4e6471470d14c8344a51be07cf0a4c718eba8ae9d021a3f2f293R237